Formula/qpid-proton.rb

```diff
@@ -0,0 +1,60 @@
+class QpidProton < Formula
+  desc "High-performance, lightweight AMQP 1.0 messaging library."
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www-us.apache.org/dist/qpid/proton/0.17.0/qpid-proton-0.17.0.tar.gz"
```

Comment:
Can this use the generic Apache mirror rather than hardcoding a specific one, such as:
```ruby
url "https://www.apache.org/dyn/closer.lua?path=spark/spark-2.2.0/spark-2.2.0-bin-hadoop2.7.tgz"
```
?